sympy/printing/maple.py

```diff
@@ -0,0 +1,167 @@
+"""
+Maple code printer
+
+The MapleCodePrinter converts single sympy expressions into single
+Maple expressions, using the functions defined in the Maple objects where possible.
+
+
+FIXME: This module is still under actively developed. Some functions may be not completed.
+"""
+
+from __future__ import print_function, division
+
+from sympy.codegen.ast import Assignment
+from sympy.core import S
+from sympy.core.compatibility import string_types, range
+from sympy.printing.codeprinter import CodePrinter
+from sympy.printing.precedence import precedence, PRECEDENCE
+
+_known_func_same_name = [
+    'sin', 'cos', 'tan', 'sec', 'csc', 'cot', 'sinh', 'cosh', 'tanh', 'sech',
+    'csch', 'coth', 'exp', 'floor'
+]
+
+known_functions = {
+    # Sympy -> Maple
+    # FIXME: maybe need added
+    'Abs': 'abs',
+    'log': 'ln',
+    'asin': 'arcsin',
+    'acos': 'arccos',
+    'atan': 'arctan',
+    'asec': 'arcsec',
+    'acsc': 'arccsc',
+    'acot': 'arccot',
+    'asinh': 'arcsinh',
+    'acosh': 'arccosh',
+    'atanh': 'arctanh',
+    'asech': 'arcsech',
+    'acsch': 'arccsch',
+    'acoth': 'arccoth',
+    'ceiling': 'ceil',
+}
+
+atomic_expr = {
+    # Sympy -> Maple
+    S('pi'): 'Pi',
```

Comment:
You can do S.Pi, S.I, and S.Infinity here to directly access the constants. (S('pi') converts the string and so on.)

Btw, these are the constants (and domains/sets, which are not AtomicExpr) available in S:

```
['Catalan',
 'ComplexInfinity',
 'Complexes',
 'EulerGamma',
 'Exp1',
 'GoldenRatio',
 'Half',
 'IdentityFunction',
 'ImaginaryUnit',
 'Infinity',
 'Integers',
 'NaN',
 'Naturals',
 'Naturals0',
 'NegativeInfinity',
 'NegativeOne',
 'One',
 'Pi',
 'Rationals',
 'Reals',
 'TribonacciConstant',
 'UniversalSet',
 'Zero',
 'false',
 'true']
```